README.md

```diff
@@ -1,15 +1,33 @@
-# doc.bytearena.com
+`# doc.bytearena.com
```

Comment:
This change seems not intentional?

Reply:
yes, sorry
I fixed it in a later commit